llama.cpp

```diff
@@ -6116,6 +6116,7 @@ static bool llm_load_tensors(
                 mlock_buf->init   (ggml_backend_buffer_get_base(buf));
                 mlock_buf->grow_to(ggml_backend_buffer_get_size(buf));
             }
+            bufs.reserve(ml.files.size());
```

Comment:
Already reserved on line 6060

Reply:
fix it https://github.com/ggerganov/llama.cpp/pull/7272/commits/4ee29e5e1caf29e1bc7b094226faa890ae0e98d6